src/background/actions/assessment-action-creator.ts

```diff
@@ -246,6 +255,14 @@ export class AssessmentActionCreator {
         );
     };
 
+    private onExpandTestNav = (payload: ExpandTestNavPayload): void => {
+        this.assessmentActions.setExpandedTestNav.invoke(payload);
+    };
+
+    private onCollapseTestNav = (): void => {
+        this.assessmentActions.setExpandedTestNav.invoke({ selectedTest: null });
```

Comment:
nit: I'm not sure how I feel about this ... to have to specify null, seems a bit odd. Ideally there's intention here to "collapseExpandedTestNav" or something ...